### PR TITLE
Do not error on installing duplicate shutdown handler

### PR DESCRIPTION
**Pull request by mheon**
Installing a duplicate shutdown handler fails, but if a handler with the same name is already present, we should be set to go. There's no reason to print a user-facing error about it.

This comes up almost nowhere because Podman never makes more than one Libpod runtime, but there is one exception (`system reset`) and the error messages, while harmless, were making people very confused (we got several bug reports that `system reset` was nonfunctional).

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@containers/podman-maintainers PTAL

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by jwhonce** [comment]
/hold
/lgtm

pending tests

**Comment by TomSweeneyRedHat** [comment]
LGTM
/hold cancel

**Pull request merged**
